### PR TITLE
add links to hombrew installation, curl, wget and qleetctl Unisntall section

### PR DESCRIPTION
**Pull request by AndriyKalashnykov**
Signed-off-by: Andriy Kalashnykov <AndriyKalashnykov@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,75 @@
-# QleetOS User Documentation
+# qleetctl
```
Comment (lander2k2) [inline README.md:1]: This repo is for user documentation - not qleetctl.  Not sure why we want to change the title here

File: README.md [hunk 2]
```diff
@@ -1,18 +1,75 @@
...
+* [curl](https://help.ubidots.com/en/articles/2165289-learn-how-to-install-run-curl-on-windows-macosx-linux)
+* [wget](https://www.gnu.org/software/wget/)
+* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [homebrew](https://brew.sh/) MacOS users
 
-Prerequisistes:
+## Install
 
-* [mkdocs](https://www.mkdocs.org/)
```
Comment (lander2k2) [inline README.md:9]: The mkdocs tool is necessary to run the local server when writing docs.  Why remove it from requirements?

File: README.md [hunk 3]
```diff
@@ -1,18 +1,75 @@
...
+Run `release` target
 ```bash
-mkdocs serve
+make release
 ```
 
-View the site at [http://127.0.0.1:8000](http://127.0.0.1:8000/)
+## Quickstart
 
+Install the QleetOS control plane:
```
Comment (lander2k2) [inline README.md:48]: I'm a bit confused about why we're adding developer docs for installing Qleet here

File: README.md [hunk 4]
```diff
@@ -1,18 +1,75 @@
...
+Remove the QleetOS control plane:
+
+```bash
+qleetctl uninstall
+```
+
+### Help
+
+```text
+$ make
```
Comment (lander2k2) [inline README.md:63]: These make targets seem like they're for the qleetctl repo.  We don't even have a Makefile in this repo yet :/

**Review by lander2k2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AndriyKalashnykov) [reply README.md:9]: not sure why it was there to begin with :)

File: README.md (hunk 4 above)
Comment (AndriyKalashnykov) [reply README.md:63]: i think you copied README from qleetctrl repo :)

File: README.md (hunk 3 above)
Comment (AndriyKalashnykov) [reply README.md:48]: prob you meant to change it later?

File: README.md (hunk 4 above)
Comment (lander2k2) [reply README.md:63]: Who did?  You might want to check `git blame`

**Pull request merged**